task_DEV_4/task_DEV_4/Rectangle.cs

```diff
@@ -1,40 +1,30 @@
 ﻿namespace task_DEV_4
 {
-    class Rectangle
+    class Rectangle : Square
     {
-        public int height;
-        public int width;
+        public double Height { get; set; }
+        public double Width { get; set; }        
```

Comment:
Why do you have here two properties if one is existed already in Square?

Reply:
Done